examples/common/aliases.ts

```diff
@@ -7,25 +7,29 @@ import { AccountAddress, getAlias, isAlias } from '@concordium/node-sdk';
  * (0 <= counter < 2^24) to determine which alias to return.
  */
 
-const accountAddress = new AccountAddress(
-    '3sAHwfehRNEnXk28W7A3XB3GzyBiuQkXLNRmDwDGPUe8JsoAcU'
-);
-const seperateAccount = new AccountAddress(
-    '4ZJBYQbVp3zVZyjCXfZAAYBVkJMyVj8UKUNj9ox5YqTCBdBq2M'
-);
+(async () => {
```

Comment:
True, blunder on my end, I just wanted consistent tabulation in the examples, but the correct way to have done that would be to just have the block I guess.

The reasoning is that since we include them in the documentation the non-asynchronous examples should ideally have the same tabulation on the typedoc documentation page.

Reply:
I see.. I think my opinion is we keep the code as simple and to the point as possible, so I'd prefer if we remove the unnecessary IIFE wrappers where possible.